docs/architecture/adr-044-lite-client-with-weak-subjectivity.md

```diff
@@ -23,7 +25,8 @@ synchronize a lite client without downloading the intervening headers.
 
 Light clients (and full nodes) operating in the Proof Of Stake context need a
 trusted block height from a trusted source that is no older than 1 unbonding
-window plus a configurable evidence submission synchrony bound. This is called “weak subjectivity”.
+window plus a configurable evidence submission synchrony bound. This is called
```

Comment:
We assume normally smaller trusted period tp for the lite client such that tp + evidence submission bound < unbonding period.